src/pages/homePage.js

```diff
@@ -1,3 +1,29 @@
-export default function HomePage() {
-  return <>HomePAge</>;
+import styled from "styled-components";
+export default function HomePage({ inputValue, selectValue }) {
+  return (
+    <>
+      <SectionInput>
+        <div>
```

Comment:
the `div` is not necessary, as you styled the `<SectionInput>`. You can remove the div and get rid of one layer of nesting.

Reply:
also already done, thanks